packages/base-extension/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "base-extension",
+  "version": "1.0.0",
```

Comment:
It may be nice to use the same version number for all packages in this repo. In addition to that, it would be great to use one centralized publishing logic like `lerna publish`. So we can easily publish all packages with one tag. This is not the scope of this PR. However, I wanted to mention it.

Reply:
> It may be nice to use the same version number for all packages in this repo. 

On the contrary, these packages should have their independent versions. Most of the time we update all extensions together but it doesn't have to be that way. There are some cases where we update only a particular extension or a package. But I am also curious why did you think that would be nice, any specific use case you have in mind? 

> In addition to that, it would be great to use one centralized publishing logic like `lerna publish`. So we can easily publish all packages with one tag. This is not the scope of this PR. However, I wanted to mention it.

This is possible via `lerna version`, which creates one tag per package to be published/updated. I think we need to document this and/or add an npm script to make it available within the repo.